pkg/cri/config/config_windows.go

```diff
@@ -23,10 +23,28 @@ import (
 
 	"github.com/containerd/containerd"
 	"github.com/containerd/containerd/pkg/cri/streaming"
+	"github.com/pelletier/go-toml"
 )
 
 // DefaultConfig returns default configurations of cri plugin.
 func DefaultConfig() PluginConfig {
+	hypervisorShimOpts := `
+		# SamdboxPlatform is a CRI setting that specifies the platform
+		# architecture for all sandbox's in this runtime. Values are
+		# 'windows/amd64' and 'linux/amd64'.
+		SandboxPlatform = "windows/amd64"
+
+		# SandboxIsolation is a CRI setting that specifies the isolation level of
+		# the sandbox. For Windows runtime PROCESS and HYPERVISOR are valid. For
+		# LCOW only HYPERVISOR is valid and default if omitted.
+		SandboxIsolation = 1
+
+		# ScaleCpuLimitsToSandbox indicates that container CPU limits should
+		# be adjusted to account for the difference in number of cores between the
+		# host and UVM.
+		ScaleCpuLimitsToSandbox = true
+	`
+	tree, _ := toml.Load(hypervisorShimOpts)
```

Comment:
Ignoring error here is probably bad, since if parsing fails we will just silently pass no opts (or bad opts). If we switch to a map like suggested below we can avoid this too.

Reply:
Answered on the reason for the toml parse above but I don't think it's worth to keep so changing